### PR TITLE
Feature/about: Created README

### PR DESCRIPTION
**Pull request by phoelapyae23**
Wrote about Git.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+#Git - Version Control System
```
Comment (phyoko72) [inline README.md:1]: Write more text.

**Review by phyoko72 — CHANGES_REQUESTED** [review]
Good job. Write more text and keep going!

File: README.md (hunk 1 above)
Comment (phoelapyae23) [reply README.md:1]: Thanks, I will do.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+#A new line of code.
+#Another line of code.
```
Comment (phyoko72) [inline README.md:3]: Ok.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#Git - Version Control System
+#A new line of code.
```
Comment (phyoko72) [inline README.md:2]: Nice.

**Review by phyoko72 — APPROVED** [review]
Great. Go ahead.

**Comment by phoelapyae23** [comment]
Please merge if you ok.

**Pull request merged**